### PR TITLE
Implement global values for image registry and PSPs

### PR DESCRIPTION
**Pull request by fhielpos**
This PR updates the values so we can use global for enabling PSPs and setting the image registry

### CODE REVIEW

**Review by stone-z — APPROVED** [review]
Would be pretty funny if we missed this

File: helm/exception-recommender/templates/crd-install/crd-psp.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{- if .Values.crds.install }}
-{{- if .Values.psp.enabled }}
+{{- if not .Values.global.podSecurityStandards.enabled }}
```
Comment (stone-z) [inline helm/exception-recommender/templates/crd-install/crd-psp.yaml:2]:
```suggestion
{{- if not .Values.global.podSecurityStandards.enforced }}
```

Comment (fhielpos) [reply helm/exception-recommender/templates/crd-install/crd-psp.yaml:2]: Woops

**Comment by fhielpos** [comment]
> Would be pretty funny if we missed this

I almost did 😂

**Pull request merged**